cmd/rtrmon/rtrmon.go

```diff
@@ -386,7 +393,7 @@ func BuildNewVrpMap(log *log.Entry, currentVrps VRPMap, newVrps []prefixfile.VRP
 
 		firstSeen := tCurrentUpdate
 		currentEntry, ok := currentVrps[key]
-		if ok {
+		if ok && currentEntry.Visible {
```

Comment:
Let's add a comment? I think this updates `lastSeen` to the current time, when an item was in grace period, but is seen in the real source again?